documentation/src/paradox/WSQuickStart.md

```diff
@@ -0,0 +1,119 @@
+# Quick Start to WS SSL
+
+This section is for people who need to connect to a remote web service
+over HTTPS, and don't want to read through the entire manual. If you
+need to set up a web service or configure client authentication, please
+proceed to the @ref:[next section](CertificateGeneration.md).
+
+## Connecting to a Remote Server over HTTPS
+
+If the remote server is using a certificate that is signed by a well
+known certificate authority, then WS should work out of the box without
+any additional configuration. You're done!
+
+If the web service is not using a well known certificate authority, then
+it is using either a private CA or a self-signed certificate. You can
+determine this easily by using curl:
+
+```
+curl https://financialcryptography.com # uses cacert.org as a CA
+```
```

Comment:
![ssl](https://cloud.githubusercontent.com/assets/422086/20477999/fc7240a0-afe0-11e6-803e-2ed8e9540f94.png)

Syntax highlighting thinks that `//` is a start of a comment. Can we adjust that by giving a hint after ```?